src/gui/attribute_table.py

```diff
@@ -21,6 +21,48 @@ def refresh(table, layerName):
 
         view = table.findChild(QTableView, "mTableView")
         view.resizeColumnsToContents()
+        for i in range(view.model().columnCount()):
+            if view.model().headerData(i, 1) in [
+                "side",
+                "min",
+                "max",
+                "age",
+                "angle",
+                "status",
+            ]:
+                view.setColumnWidth(
+                    i,
+                    QLabel("")
+                    .fontMetrics()
+                    .boundingRect(view.model().headerData(i, 1))
+                    .width(),
+                )
+            elif view.model().headerData(i, 1) in [
+                "podSize",
+                "distance",
+                "angle",
+                "direction",
+                "visibility",
+                "subjectiveMam",
+                "subjectiveBirds",
+                "nObservers",
+                "glareSever",
+                "seaState",
+                "windDir",
+                "windForce",
+                "swellDir",
+                "swellHeight",
+                "glareFrom",
+                "glareTo",
+            ]:
+                view.setColumnWidth(
+                    i,
+                    QLabel("")
+                    .fontMetrics()
+                    .boundingRect(view.model().headerData(i, 1))
+                    .width()
+                    + 5,
+                )
```

Comment:
It's definitely this part of the code which causes the segfaults. Can we remove it from the PR for now?

Reply:
All of it or just the `elif` part ?

Reply:
It's actually the `view.setColumnWidth(...)` which causes the segfault, so all of it.

Reply:
ok it's done.

Just to know, which version of Qt do you use ?

Reply:
![image](https://user-images.githubusercontent.com/9266424/203819321-3b17dbd0-96da-477b-bcb7-d8a425850d50.png)